common/features/config/networks/static/reducer.ts

```diff
@@ -698,6 +699,58 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
       max: 60,
       initial: 20
     }
+  },
+
+  ETI: {
+    id: 'ETI',
+    name: 'ETI',
+    unit: 'ETI',
+    chainId: 101,
+    isCustom: false,
+    color: '#3560bf',
+    blockExplorer: makeExplorer({
+      name: 'EtherInc Explorer',
+      origin: 'https://explorer.einc.io'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: ETI_DEFAULT,
+      [SecureWalletName.SAFE_T]: ETI_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: ETI_DEFAULT,

```

Comment:
Please link to your Ledger PR, or remove the Ledger support if it isn't actually supported.